integration_test/third_party_apps_data/applications/flink/metadata.yaml

```diff
@@ -0,0 +1,36 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/flink"
```

Comment:
This shouldn't be set as this is a 404 non existent link.

When google technical writer has the documentation, we will set this public_url in post agent release jobs :)

Reply:
Oh great catch! Will fix this as part of the flink metrics PR. Also made a pass through the other PRs that added this and commented on them too.